processing/src/main/java/io/druid/query/metadata/metadata/SegmentAnalysis.java

```diff
@@ -38,6 +38,7 @@
   private final long numRows;
   private final Map<String, AggregatorFactory> aggregators;
   private final QueryGranularity queryGranularity;
+  private final Boolean rollup;
```

Comment:
I will modify related code to this logic: when merge SegmentAnalysis, false&false=false, true&true=true, true&false=null
so I need it be Boolean instead of boolean